src/ResourceManagement/AnalysisServices/AnalysisServices.Tests/ScenarioTests/ServerOperationsTests.cs

```diff
@@ -0,0 +1,163 @@
+﻿
+//
+// Copyright (c) Microsoft.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+using AnalysisServices.Tests.Helpers;
+using Microsoft.Azure;
+using Microsoft.Azure.Management.Analysis;
+using Microsoft.Azure.Management.Analysis.Models;
+using Microsoft.Azure.Test;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using Xunit;
+
+namespace AnalysisServices.Tests.ScenarioTests
+{
+    public class ServerOperationsTests : TestBase
+    {
+        [Fact]
+        public void CreateGetUpdateDeleteTest()
```

Comment:
>CreateGetUpdateDeleteTest [](start = 20, length = 25)

Can this be split into multiple smaller tests? If possible.
For e.g. can the update/delete be separate tests reusing the create from a common code.
this is just a guidance, because in case if the test failed, you won't know if the create failed or update or deleting the account failed.